### DISCUSSION TITLE
5.0.0 release

### DISCUSSION THREAD

**Discussion by luisgmgouveia** [FirelyTeam/firely-net-sdk#2381] [General]
Hi guys,

I'm still using the 4.3.0 version (september 22). As I need the MedicinalProductDefinition.ComprisedOf new reference, I was wondering when will the 5.0.0 version be released.

I see the 5.0.0-beta3 was released yesterday. Can you forecast in how many weeks the official 5.0.0 version will be released?

Thank you!

**Comment by marcovisserFurore** [comment]
The official release of version 5.0.0 took place on February 17th, 2023. Since then, we have already rolled out the third minor release (5.3.0).
